router_macro_derive/src/lib.rs

```diff
@@ -13,7 +13,7 @@ extern crate proc_macro_error;
 use crate::{root::get_default_route, routing::routing_variant_snippets};
 use proc_macro::TokenStream;
 
-use crate::{init::module_init_snippets, modules::modules_path, view::modules_view_snippets};
```

Comment:
managed to clean up a repeated_module_names lint. Now uses `modules::path`. The proc_macro is still `modules_path = ...`